src/configure/templates/nodejsWindowsFunctionApp.yml

```diff
@@ -0,0 +1,81 @@
+# Node.js Function App to Linux on Azure
+# Build a Node.js function app and deploy it to Azure as a Linux function app.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- {{{ sourceRepository.branch }}}
+
+variables:
+  # Azure Resource Manager connection
+  azureSubscription: '{{{ targetResource.serviceConnectionId }}}'
+
+  # Web app name
+  functionAppName: '{{{ targetResource.resource.name }}}'
+
+  # Environment name
+  environmentName: '{{ pipelineParameters.environment }}'
```

Comment:
this is same as targetResource.resource.name and can be used here

Reply:
Yes but what if we need custom environment name? We're looking for it to be extensible right

Reply:
you're right, but we should solve it differently. By getting a list of required inputs from template and then getting those inputs.

Right now adding environment into pipelineParameters, doesn't seem right.
For now we can use a hard coded, environment: development and then later when we deploy to slots and have multiple environments. We can choose to counter it then

Reply:
using hardcoded environment can polute existing ones, if there's already the environment 'development' for example. environment is a pipeline parameter, why'd you feel it is not right? I think we can keep this solution for now, but like you said, if we're planning on getting list of required inputs from templates and such, we can change

Reply:
We are still not asking the user for this input ? If we deriving it from the targetResource.name, we can reuse the same. 
The same **parameterParameters** will be used for **GitHub Actions** as well, where some of this parameters are not used. 

We can introduce it later when we dig more in Azure Pipelines and start gettings required input list from templates, we could then add this or we could still reuse target resource name for environment.

Reply:
Ah it's for Github actions. I'll remove then